frontend/awx/access/organizations/Organizations.tsx

```diff
@@ -64,8 +65,8 @@ export function Organizations() {
         variant: ButtonVariant.primary,
         icon: PlusIcon,
         label: t('Create organization'),
-        href: RouteObj.CreateOrganization,
-      },
+        onClick: () => navigate(RouteObj.CreateOrganization),
+      } as IPageActionButton,
```

Comment:
The href is preferrable as a user can right click and open in new window.  Keith put that in.